tests/ui/js/storyTelling-page-storyEditTests.js

```diff
@@ -449,6 +884,10 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         jqUnit.assertEquals("The response text is as expected", expectedText, actualText);
     };
 
+    sjrk.storyTelling.page.storyEditTester.setCurrentBlock = function (testCaseHolder, currentBlock) {
+        testCaseHolder.options.members.currentBlock = currentBlock;
```

Comment:
Member options can be accessed as a direct link to the component, such as: `testCaseHolder.currentBlock = currentBlock;`.

Similarly, all appearances of `{storyEditTester}.options.members.currentBlock` at line 129, 151, 170 can be changed to  `{storyEditTester}.currentBlock`.

Also, this function can also be replaced by declarative statements, for example, line 114-116 can be changed to:
```
{
    "event": "{storyEdit}.storyEditor.events.onNewBlockTemplateRendered",
    listener: "fluid.set",
    args: ["{storyEditTester}", ["currentBlock"], "{arguments}.0"]
}
```

The same comment applies to other places where `setCurrentBlock()` is used.